sonar-python-plugin/src/main/java/org/sonar/plugins/python/IPynbSensor.java

```diff
@@ -63,9 +63,10 @@ public void describe(SensorDescriptor descriptor) {
   @Override
   public void execute(SensorContext context) {
     List<InputFile> pythonFiles = getInputFiles(context);
-    context.config().get(PYTHON_VERSION_KEY)
-      .map(PythonVersionUtils::fromString)
-      .ifPresent(ProjectPythonVersion::setCurrentVersions);
+    var pythonVersions = context.config().getStringArray(PYTHON_VERSION_KEY);
+    if (pythonVersions.length != 0) {
+      ProjectPythonVersion.setCurrentVersions(PythonVersionUtils.fromString(pythonVersions));
```

Comment:
This goes untested, yet I think we can have a (very simple) test in `IPynbSensorTest` that does something similar to what is done in `PythonSensorTest#test_python_version_parameter_no_warning` (setting the property in the context before executing the sensor test) so that this would be well tested. (It would not exactly be the same test though, as there is no warning on SonarLint for missing version property).

Note that the missing test probably simply went unnoticed due to the optional chain that was used before.